examples/display_data.py

```diff
@@ -12,15 +12,39 @@
 """
 
 from parlai.core.params import ParlaiParser
+from parlai.core.agents import Agent
 from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
 from parlai.core.worlds import create_task
 
 import random
 
+
+class RepeatTextAgent(Agent):
```

Comment:
if we need another agent we'd probably put it elsewhere, or extend RepeatLabelAgent somehow ... still trying to understand exactly what you want it to do ..